### PR TITLE
Added case study in the csv download

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: supplier/admin.py [hunk 1]
```diff
@@ -47,11 +48,19 @@ def download_csv(self, request, queryset):
...
         fieldnames = sorted(fieldnames)
 
-        suppliers = queryset.select_related('company').all().values(
-            *fieldnames
-        )
+        suppliers = queryset.select_related('company').all().annotate(
+            company__has_case_study=Case(
+                When(
+                    company__supplier_case_studies__isnull=False,
+                     then=Value(True)
```
Comment (richtier) [inline supplier/admin.py]: funky indentation here

File: supplier/admin.py [hunk 2]
```diff
@@ -47,11 +48,19 @@ def download_csv(self, request, queryset):
...
+                ),
+                default=Value(False),
```
Comment (sdonk) [inline supplier/admin.py]: we probably don't need default

File: supplier/admin.py (hunk 1 above)
Comment (sdonk) [inline supplier/admin.py]: Flake8 is happy ;)

Comment (sdonk) [inline supplier/admin.py]: actually it's not on circleci

File: supplier/admin.py (hunk 2 above)
Comment (sdonk) [inline supplier/admin.py]: We do! Default is empty otherwise

**Pull request merged**